dart/lib/src/scope.dart

```diff
@@ -0,0 +1,139 @@
+import 'dart:collection';
+
+import 'protocol.dart';
+import 'sentry_options.dart';
+
+/// Scope data to be sent with the event
+class Scope {
+  /// How important this event is.
+  SeverityLevel _level;
+
+  SeverityLevel get level => _level;
+
+  set level(SeverityLevel level) {
+    _level = level;
+  }
```

Comment:
maybe we should getters and setters only for List/Maps so they cannot mutate the whole thing?
https://dart.dev/guides/language/effective-dart/usage#dont-wrap-a-field-in-a-getter-and-setter-unnecessarily

Reply:
Makes sense to me!